### ISSUE TITLE
Add Notes on Pointers?

### ISSUE THREAD

**Issue by kkukshtel** [aelij/spanify#2]
Hello! Great repo and think it's a great reference for lower-level memory management in C#.

One thing I would love to see though is some mention of pointers, primarily for interop use cases. Things like Span<T> are great, but the type is still managed so the pointer to a Span<T> isn't stable for interop use.

[I wrote a blog post](https://blog.kylekukshtel.com/csharp-lowlevel-memory-pinvoke-span-blittable-bindings) on some of these ideas, in part because it didn't feel like people were really talking about real interop usecases around this, and the other big note that things like Span<T> on really work in locally scoped functions.

Basically, I think this repo is a great resource, and would love to read all that you know on how these new Memory types work for use cases outside of method scope + interop.

**Comment by aelij** [comment]
Hi Kyle! Thanks for the suggestion, the blog post is an interesting read.
I'm currently focused on performance related optimizations for C# rather than interop scenarios.
I'll keep this in mind for the future though.
